Content.Server/Speech/EntitySystems/ReplacementAccentSystem.cs

```diff
@@ -80,6 +89,8 @@ public string ApplyReplacements(string message, string accent)
 
                     // In-place replace the match with the transformed capitalization replacement
                     message = message.Remove(match.Index, match.Length).Insert(match.Index, replacement);
+                    string mask = new string('_', match.Length);
+                    maskMessage = message.Remove(match.Index, match.Length).Insert(match.Index, mask);
```

Comment:
am I misunderstanding something or `maskMessage` is just useless here? It's not being return nor it's used for anything (not including when you just replace `message` usage with `maskMessage`)

Reply:
> regex searches not by the final message, but by this mask, from which the replaced words are erased

but maskMessage is never returned ain't it?

Reply:
Its used for search matches

Reply:
if you don’t use such a buffer, but replace it one by one, it will turn out like this: